packages/lighthouse-service/.env.example

```diff
@@ -12,3 +12,6 @@ GITLAB_URL=
 
 API_GATEWAY=
 GATEWAY_AUTH_TOKEN=
+
+## Lighthouse Config
+LHCI_SERVER_URL=
```

Comment:
@riginoommen  actually used the env `SERVER_BASE_URL` for pointing to LH server, which now i think yes can be confusing..

But now both means one thing so one need to be dropped

Reply:
I have dropped LHCI_SERVER_URL.